src/cmdlinetest/test_help.t

```diff
@@ -37,6 +37,11 @@ Usage:
                           used multiple times
       -P <property>=<value>
                           Set/ override a property value
+      -x <task>, --exclude=<task>
+                          Exclude optional task dependencies
+      --force-exclude=<task>
+                          Force exclude any task dependencies (dangerous, may
```

Comment:
```
(current)bash-3.2$ ./build.py analyze --force-exclude=analyze
PyBuilder version ${version}
Build started at 2015-09-07 03:40:09
------------------------------------------------------------
[INFO]  Building PyBuilder version 0.10.63
[INFO]  Executing build in /Users/arcivanov/Documents/src/arcivanov/pybuilder
[INFO]  Going to execute task analyze
[INFO]  Executing flake8 on project sources.
------------------------------------------------------------
BUILD SUCCESSFUL
------------------------------------------------------------
Build Summary
             Project: PyBuilder
             Version: 0.10.63
      Base directory: /Users/arcivanov/Documents/src/arcivanov/pybuilder
        Environments: 
               Tasks: analyze [1089 ms]
Build finished at 2015-09-07 03:40:10
Build took 1 seconds (1108 ms)
(current)bash-3.2$ 
```

Hmm, good point. And I'll reword, let me see if I can change this to make sense.